src/styles/App.scss

```diff
@@ -14,37 +14,31 @@ body {
 }
 
 #catalog-image-list {
-  @include mdc-image-list-aspect(1.25); // Images are 5:4
-  @include mdc-image-list-standard-columns(5, 1px);
+  @include mdc-image-list-aspect(1); // Images are square
+  @include mdc-image-list-standard-columns(5, 6px);
 
   max-width: 1000px;
-  background-color: #e8eaed;
-  padding-bottom: 1px;
 }
 
 @media (max-width: map-get($breakpoints, tablet)) {
   #catalog-image-list {
-    @include mdc-image-list-standard-columns(3, 1px);
+    @include mdc-image-list-standard-columns(3, 6px);
   }
 }
 
 @media (max-width: map-get($breakpoints, mobile)) {
   #catalog-image-list {
-    @include mdc-image-list-standard-columns(2, 1px);
+    @include mdc-image-list-standard-columns(2, 6px);
   }
 }
-
-.catalog-image-list-item {
-  text-decoration: none;
-  background-color: white;
-}
-
-.catalog-image-list-supporting {
-  padding-left: 20px;
+.catalog-image-list-item-container {
+  background-color: #f5f5f5;
```

Comment:
These hardcoded values will probably used somewhere else. Want to make like a colors.scss file with these values? Or do you have any other ideas?

Reply:
Updated it to use the Material color palette.

Reply:
Nice! Looks good